### ISSUE TITLE
sys-kernel/genkernel-next deprecation

### ISSUE THREAD

**Issue by frunzales** [sakaki-/sakaki-tools#22]
The current genkernel-next package is being deprecated in the official Gentoo portage tree:

[https://bugs.gentoo.org/719968](https://bugs.gentoo.org/719968)

sakaki-tools:sys-kernel/buildkernel depends on sys-kernel/genkernel-next as a runtime dependency:

```
RDEPEND="${DEPEND}
	>=sys-libs/ncurses-5.9-r2
	>=virtual/linux-sources-3
	>=app-crypt/sbsigntools-0.6-r1
	plymouth? ( >=sys-boot/plymouth-0.8.8-r4[gdm,libkms,pango] )
	>=sys-kernel/genkernel-next-58[cryptsetup,gpg,plymouth?]
	=app-crypt/staticgpg-1.4.16-r1
	>=sys-boot/efibootmgr-0.5.4-r1"
```

Are there plans to support the Gentoo-supplied sys-kernel/genkernel?

**Comment by sakaki-** [comment]
If `genkernel-next` doesn't get proxy-maintained (per [the bug](https://bugs.gentoo.org/719968#c13)), then yes I will have to migrate.

Best, sakaki

**Comment by TheEnthralled** [comment]
Funny enough I was following the guide right as this happened!

Some in the bug thread say that the bug may only affect version 70, so is it possible to just unmask it and continue with the installation?

**Comment by sakaki-** [comment]
Yes, for now creating e.g. `/etc/portage/package.unmask/genkernel-next` with the contents:
```
~sys-kernel/genkernel-next-69
```
should work. I'll add this to the guide now as a band-aid until this is resolved one way or other upstream.

**Comment by sakaki-** [comment]
Guide [updated](https://wiki.gentoo.org/index.php?title=Sakaki%27s_EFI_Install_Guide%2FBuilding_the_Gentoo_Base_System_Minus_Kernel&type=revision&diff=881706&oldid=881115).

**Comment by necrose99** [comment]
so far sabayon is preparing to sawp base to funtoo.  as funtoo has a more backdated but stable toolbase , and the option of newer tools.

**Comment by sakaki-** [comment]
Now that `genkernel-next` has been treecleaned, I've added v69 to this overlay, and updated the guide accordingly. Obviously not  a long-term fix, but will allow `buildkernel` to keep working for now.

**Comment by donbcd** [comment]
I am using a recent kernel (5.8+) because of hardware issues. With these recent kernel versions + genkernel-next-69 the kernel does not boot. With genkernel-next-70 booting the recent kernels worked again, buildkernel worked fine, too.
Is it possible to offer genkernel-next-70, too, until the overall issue is solved?

**Comment by yantar92** [comment]
> Now that genkernel-next has been treecleaned, I've added v69 to this overlay, and updated the guide accordingly. Obviously not a long-term fix, but will allow buildkernel to keep working for now.

I am currently installing fresh Gentoo using the guide. Just adding `genkernel-next-69` is not enough. It does not solve the issue with old version of busybox used by genkernel (which is not compatible with latest stable glibc) [1].

I had to add `BUSYBOX_VER="1.32.0"` to genkernel.conf and create a symlink to that version's source tarball (note that 1.32.0 is `~amd64` for now):

```` bash
ln -s /var/cache/distfiles/busybox-1.32.0.tar.bz2 /var/lib/genkernel/src/
````

[1] https://bugs.gentoo.org/719968

**Comment by sakaki-** [comment]
31 Oct 2020: sadly, due to legal obligations arising from a recent change in my 'real world' job, I must announce I am **standing down as maintainer of this project with immediate effect**. For the meantime, I will leave the repo up (for historical interest, and since the ebuilds etc. may be of use to others); however, I plan no further updates, nor will I be accepting / actioning further pull requests or bug reports from this point. Email requests for support will also have to be politely declined, so, **please treat this as an effective EOL notice**.<br><br>For further details, please see my post [here](https://forums.gentoo.org/viewtopic-p-8522963.html#8522963).<br><br>If you have used my [EFI Guide](https://wiki.gentoo.org/wiki/User:Sakaki/Sakaki%27s_EFI_Install_Guide) (and this repo) to install your PC-based Gentoo system, it *should* still continue to work for some time, **but** you should now take steps to migrate to a baseline [Gentoo Handbook](https://wiki.gentoo.org/wiki/Handbook:AMD64) install (since the underlying tools, such as [`buildkernel`](https://github.com/sakaki-/buildkernel), will also now no longer be supported and may eventually fail as more modern kernels etc. are released).<br><br>With sincere apologies, sakaki ><
